src/commands/remember/basicRemember.js

```diff
@@ -13,7 +14,7 @@ module.exports = {
             type: ApplicationCommandOptionType.String,
         },
         {
-            name: 'search-specific-channel',
+            name: 'channel-id',
             description: 'Searches the give channel for the message.',
```

Comment:
Should be changed to reference the id. Something like `The id of the channel that is being searched`